cmd/beacon-chain/main.go

```diff
@@ -138,6 +139,9 @@ var appFlags = []cli.Flag{
 	flags.JwtId,
 	storage.BlobStoragePathFlag,
 	storage.BlobRetentionEpochFlag,
+	backfill.EnableExperimentalBackfill,
```

Comment:
can you add this as a feature flag instead ? since we eventually will deprecate it once the feature is the default.